modules/client-configuration/pages/client-operations.adoc

```diff
@@ -0,0 +1,10 @@
+[[client.operations]]
+= Client Operations
+
+In addition to registering, upgrading, or deleting clients other operations can be performed.
+
+{productname} clients can be managed individually, or organised in groups using System Set Manager, System Groups or Configuration Management.
```

Comment:
```suggestion
{productname} clients can be managed individually, or organized in groups using System Set Manager, System Groups or Configuration Management.
```
I missed this British spelling also in `master`.  It would be good if you'd fix that one straight away.